base/pkg.jl

```diff
@@ -2,9 +2,9 @@
 
 module Pkg
 
-export Git, Dir, Types, Reqs, Cache, Read, Query, Resolve, Write, Entry, Git
+export Dir, Types, Reqs, Cache, Read, Query, Resolve, Write, Entry, Git
 export dir, init, rm, add, available, installed, status, clone, checkout,
-       update, resolve, test, build, free, pin, PkgError
+       update, resolve, test, build, free, pin, PkgError, set_git_proto
```

Comment:
Ack, that is really embarrassingly stupid! Sorry to keep wasting your time on such things.